triframe/phases/actor.py

```diff
@@ -28,10 +28,14 @@
 )
 from utils.logging import log_warning
 from utils.phase_utils import (
+    add_empty_user_turn,
     add_usage_request,
+    get_thinking_block,
     run_phase,
 )
 
+CLAUDE_THINKING_MODELS = ["claude-3-7-sonnet-20250219"]
```

Comment:
Third place where this value is hard-coded. It could at least be a shared constant, right? Though even better IMO  would be to make it part of the agent settings or something.